.github/workflows/wpt.yml

```diff
@@ -57,7 +57,9 @@ jobs:
       - name: Prepare Pages
         run: |
           mkdir -p out/site
+          cp out/wptreport.json out/site/wptreport.json
```

Comment:
Not strictly required, but this lets us more easily test the HTML formatter locally against the latest results (without having to rerun the tests or having to look for the latest artifact download link).

Reply:
"1 conversation must be resolved before merging."